lib/cog/commands/filter.ex

```diff
@@ -55,9 +55,9 @@ defmodule Cog.Commands.Filter do
     do: state
 
   defp validate_matches(state, matches) do
-    case String.valid?(matches) do
-      true -> %{state | match: compile_regex(matches)}
-      false -> add_errors(state, :bad_match)
+    case Regex.compile(matches) do
```

Comment:
Are we compiling the regex twice now? I know this also happens in `compile_regex` after some manipulating.